WalletWasabi.Gui/packages.lock.json

```diff
@@ -2272,8 +2272,8 @@
         "dependencies": {
           "Microsoft.AspNetCore.Mvc.NewtonsoftJson": "3.1.1",
           "Microsoft.Win32.Registry": "4.7.0",
-          "NBitcoin": "5.0.60",
-          "NBitcoin.Secp256k1": "1.0.8"
+          "NBitcoin": "5.0.73",
```

Comment:
@lontivero does this can have a side effect for other classes?

Reply:
I don't think so, @molnard. There are many changes but most of them I think are covered by unit tests

Some changes:

*    it includes the new Signet network so, we should take a look at the network selection code
*    it includes the new bech32m encoder (it modifies the existing bech32 encoder)
*    includes output descriptors
*    includes bip155
*    includes fixes for keypaths
*    includes changes to support bitcoin 0.21.0
*    includes changes to support taproot (lots of things, schnorr signatures for example)
*    updates the default hadcoded nodes list (that's good!)
*    includes many new rpc methods

Reply:
I tested this PR for 12 hours, works well.